examples/llm/tech_qa.py

```diff
@@ -139,9 +139,22 @@ def get_data():
     with open('train.json') as file:
         json_obj = json.load(file)
     text_contexts = []
-    for file_path in glob(f"corpus/*"):
-        with open(file_path, "r+") as f:
-            text_contexts.append(f.read())
+
+    # Read corpus data. Prefer *.json files, fall back to txt files.
+    file_paths = glob(f"corpus/*.json")
+    if len(file_paths) > 0:
+        for file_path in file_paths:
+            with open(file_path, "r+") as f:
+                data = json.load(f)
+            doc_type = data[0]["document_type"]
+            if doc_type != "text":
```

Comment:
please add comments about what the other types are (pdf i assume? maybe others, would be good to add)
otherwise LGTM

Reply:
and i assume you tested this on the json file examples we have?